packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -98,9 +98,13 @@ function graphql() {
   )
 }
 
-function unstable_collectionGraphql() {
-  // TODO: Strip this out of the component and throw error if it gets called
-  return null
+function collectionGraphql() {
```

Comment:
Weirdly enough this error is thrown on all E2E tests?

Reply:
It looks like we do not remove related GraphQL queries from original files. IIRC we do this with the regular `graphql` tag in the `babel-plugin-remove-graphql-queries` plugin